user-stories/US-07-Privacy-request-capture-service.md

```diff
@@ -0,0 +1,300 @@
+**because** so I can easily identify their data
+
+### US-07-2-03
+
+**As an** Organization, using blindnet devkit, receiving Privacy Request
+
+**I want to** receive the request in the Privacy Request section of my system
+
+**because** it is more convenient for me to have all my Privacy Requests centralized
+
+### US-07-2-04
+
+**As an** Organization (using or not using blindnet devkit) receiving Privacy Request
+
+**I want to** transfer the request to another Organization if the request concerns data I transferred to another Organization or data I got from another Organization
+
+**because** I need to be compliant, if a user wants their data deleted, I have to inform the other company I transferred the user's data to or I got the user's data from, to also delete the data
+
+## 07-3 Organization using Local Privacy Request capture service
+
+### US-07-3-01
+
+**As an** Organization, using blindnet devkit, receiving Privacy Request
+
+**I want to** have an entry point for the Privacy Request service embedded on the GDPR section of my website
+
+**because** my customers might directly go to my website to make their Privacy Request, and it is easier for my customers and for me to use this service
+
+### US-07-3-02
+
+**As an** Organization, using blindnet devkit, receiving Privacy Request
+
+**I want to** receive the Privacy Requests in a format that is compatible with my system
+
+**because** otherwise I cannot use blindnet devkit to for Privacy Request capture
+
+
+### US-07-3-03
+
+**As an** Organization, using blindnet devkit
+
+**I want to** only use some of the components, a subset of the scope of blindnet devkit for things I don't have yet
+
+**because** I want to continue using my existing interface that already covers some of my compliance needs
+
+### US-07-3-04
+
+**As an** Organization, using blindnet devkit
+
+**I want to** only use some of the components, a subset of the scope of blindnet devkit for things I don't have yet
+
+**because** I want to continue using my existing system that already covers some of my compliance needs (e.g my system already allows users to log in and modify data adresses, preferences etc - on Amazon, Facebook, etc)
+
+### US-07-3-05
+
+**As an** Organization, using blindnet devkit
+
+**I want to** use the whole blindnet devkit to automate the processing of requests
+
+**because** I don't have any system that covers my compliance needs, I do everything manually
+
+### US-07-3-06
+
+**As an** Organization, using blindnet devkit
+
+**I want to** use blindnet devkit to automate the processing of requests except for "other" type of request
+
+**because** I need to process "other" type of request manually
+
+### US-07-3-07
+
+**As a** developper from an Organization, using blindnet devkit
+
+**I want to** use only the capture interface component (from the HLA) to generate a json file for a request
+
+**because** this is all I need, as I want to handle the file and import it to some other system
+
+### US-07-3-08
```

Comment:
New numbering this US is now : 
>### US-07-5-05
>
>**As a** developper from an Organization, using blindnet devkit
>
>**I want to** use the capture interface component and the data consumer interface component (from the HLA)
>
>**because** I need to generate a JSON file for the requests and also to view the requests. From there I will treat the requests manually

Let's see if we keep it or not if it doesn't add any additional info and also might be out of Capture Service Scope [cf my last comment](https://github.com/blindnet-io/product-management/pull/602#discussion_r908236556)